nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py

```diff
@@ -61,6 +61,7 @@ def complete(self, cr, uid, activity_id, context=None):
         res = super(NHClinicalPatientObservationEWS, self).complete(
             cr, uid, activity_id, context=context)
         activity_model = self.pool['nh.activity']
+        datetime_utils_model = self.pool['datetime_utils']
```

Comment:
Above you use `date_utils_pool` as the variable name for the same model. I think we should always use `pool` for stuff thats come from pools and use the fullname of the model if it's not too long: `datetime_utils_pool`.

Reply:
can change